app/_includes/prereqs/entities.md

```diff
@@ -1,8 +1,8 @@
 <details class="mb-2" markdown="1">
-  <summary class="rounded mb-0.5 bg-gray-200 p-2">Entities needed for this how-to</summary>
-
-1. Create a `deck_files` folder and add your `kong.yaml` file in it.
-1. Create a `prereqs.yaml` file in the same folder, and add the following content to it:
+  <summary class="rounded mb-0.5 bg-gray-200 p-2">Pre-configured Entities</summary>
```

Comment:
Now that we renamed these to `how-tos`, should we update the copy `For this tutorial` to something else?
there's a typo in `These entities aare`